src/main/Catalog.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import InfiniteScroll from "react-infinite-scroll-component";
+import SearchForm from "./SearchForm";
+
+const beersUrl = `https://api.punkapi.com/v2/beers`;
+
+function Catalog() {
+  const [beerList, setBeerList] = useState([]);
+  const [beerNumber, setBeerNumber] = useState(9);
+
+  useEffect(() => {
+    let xhttp = new XMLHttpRequest();
+    xhttp.onload = function () {
+      if (this.readyState === 4 && this.status === 200) {
+        setBeerList(JSON.parse(xhttp.responseText));
+      }
+    };
+    xhttp.open("GET", `${beersUrl}?per_page=${beerNumber}`, true);
+    xhttp.send(null);
+  }, [beerNumber]);
+
+  return (
+    <div className="catalog-page">
+      <SearchForm />
+      <InfiniteScroll
+        dataLength={beerList.length}
+        next={setTimeout(() => setBeerNumber(beerNumber + 9), 2500)}
+        hasMore={true || false}
+        loader={<br />}
```

Comment:
I would write or find a simple loader component to show here. Or just use a "Loading..." message to let the user know that some action is running.